lib/amazon_order/parsers/order.rb

```diff
@@ -47,6 +53,11 @@ def to_hash
           hash.merge!(products: products.map(&:to_hash))
         end
       end
+
+      def order_type
+        @node.css('#Leave-Service-Feedback_15').present? ? :service_order : :item_order
```

Comment:
I think `15` could be different by the position of order.

I guess selectors for matching would work.
https://www.w3.org/TR/selectors/#attribute-substrings

```
@node.css("[id^='Leave-Service-Feedback_']")
```

Reply:
good call. will fix.